docs/lint_errors.md

```diff
@@ -102,16 +112,66 @@ The following variables are depreciated and fail the test if they are still pres
   * The old method for specifying the minimum Nextflow version. Replaced by `manifest.nextflowVersion`
 * `params.container`
   * The old method for specifying the dockerhub container address. Replaced by `process.container`
+* `singleEnd` and `igenomesIgnore`
+  * Now using `snake_case` for all command line options
```

Comment:
We should said that they're deprecated, and now all params are in `snake_case` and maybe specify what they are now.
We did not changed all command line options, just the params, all the Nextflow CL options are still as they were.

Reply:
See https://github.com/nf-core/tools/pull/483/commits/8c6d27fdb9979b5f68036f4bf1617c3f4398d3b6